README.md

```diff
@@ -43,6 +43,6 @@ Build the Multiplayer PacMan game which would work smoothly with various fun fea
 
 ### URLS: 
 
-- Game-staging: https://5c89a982fb0eb10008c6c763--youthful-dubinsky-b46f9a.netlify.com/
+- Game-staging: https://youthful-dubinsky-b46f9a.netlify.com/
```

Comment:
Finally. It was so embarrassing.

Reply:
:fire: :rocket:

Reply:
You can create a alias for this, so that it can be remain fixed

Reply:
Previously, we had appended the commit to the URL which linked to the deploy of **that commit**. 

Now, we have removed the commit from URL. By default, the `master` is deployed to this URL. So, the link will stay fixed.